src/components/ims/ims-artifact.factory.js

```diff
@@ -264,39 +266,39 @@ export class ImsArtifactApi {
     if (entry) {
       return entry.description;
     } else {
-      return ;
```

Comment:
ok, it's easier to apply first style change and then refactor the code to easily locate the changes

maybe we could apply an auto-formatter as part of the gulp process so every-one will have automatically style applied